tests/Tests/100__deploy/130__operators/130__rhods_operator/135__rhods_operator_logs_verification.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Documentation   135 - RHODS_OPERATOR_LOGS_VERIFICATION
+...             Verify that rhods operator log is clean and doesn't contain any error regarding resource kind
+...
+...             = Variables =
+...             | Namespace                | Required |        RHODS Namespace/Project for RHODS operator POD |
+...             | REGEX_PATTERN            | Required |        Regular Expression Pattern to match the erro msg in capture log|
+
+Library        Collections
+Library        OpenShiftCLI
+Library        OperatingSystem
+Library        String
+
+*** Variables ***
+${namespace}           redhat-ods-operator
+${regex_pattern}       level=([Ee]rror).*
+
+*** Test Cases ***
+Verify RHODS Operator log
+   [Tags]  ODS-1007   Sanity
+   #Get the POD name
+   ${data}       Run keyword   OpenShiftCLI.Get   kind=Pod     namespace=${namespace}   label_selector=name=rhods-operator
+   #Capture the logs based on containers
+   ${val}        Run   oc logs --tail=1000000 ${data[0]['metadata']['name']} -n ${namespace} -c rhods-operator
```

Comment:
If i understand correctly....if you have any problem with deployer you will have problem with whole RHODS end to end installation..i think it's not needed

Reply:
mmh maybe it could be helpful to have the same check on the deployer. At least we would know without looking at it directly on OCP that there was a problem during deployment. Of course, if there's a problem with the deployment I expect to see the automated test failing. I don't know, i think it could be helpful and it has a very low cost to be added, doesn't it?
Just my opinion, I might be wrong, I would also ask for other opinions about.

Reply:
I understand and it won't much time from my end to add that code. The problem what i am thinking is what kind of error we will actually look for because we see different kind of entry in deployer like Error from server(already exists) etc.

Reply:
okay. it is something we can investigate more then for a next PR. What do you think?

Reply:
Sure